cartoframes/context.py

```diff
@@ -715,6 +715,13 @@ def query(self, query, table_name=None, decode_geom=False):
             Pandas data types are inferred from PostgreSQL data types.
             In the case of PostgreSQL date types, dates are attempted to be
             converted, but on failure a data type 'object' is used.
+
+        Examples:
+            Query a table in CARTO and write a new table that is result of query
+
+            .. code:: python
+                topten_df = cc.query('SELECT * from my_table ORDER BY value_column DESC LIMIT 10',
+                            table_name='top_ten')
```

Comment:
How about a more complex example, too, like a JOIN of a point table and a polygon table, aggregated on a value, and JOINS on `ST_Intersects(points.the_geom, polys.the_geom)`?